app/assets/javascripts/custom.coffee

```diff
@@ -3,17 +3,51 @@ toggleVideo = (state) ->
   div = document.getElementById('youvideo')
```

Comment:
You should apply separation of concern principle on this file instead of putting all the code in custom.coffee.